src/NossoMercadoLivreAPI.Service/Exceptions/AppException.cs

```diff
@@ -0,0 +1,18 @@
+﻿using System;
+
+namespace NossoMercadoLivreAPI.Service.Exceptions
+{
+    public class AppException : Exception
```

Comment:
*Usamos tudo que conhecemos que está pronto. Só fazemos código do zero se for estritamente necessário.*

Provavelmente já tem exception pronta para tudo que você vai precisar no código. Sugiro remover